src/System.Text.Encoding/tests/System.Text.Encoding.Tests.csproj

```diff
@@ -12,8 +12,10 @@
   </PropertyGroup>
   <!-- Default configurations to help VS understand the configurations -->
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
why you need to repeat adding this property for every configuration? you can just add it globally for the whole project so we don't have to duplicate it for every configuration